commit 214443fdff330629104509de9e1738aadcfe36b6
Author: Marco Martin <notmart@gmail.com>
Date:   Mon Jul 25 10:16:03 2016 +0200

    apparently containment() can be null
    
    null in some rare occasions during startup
    CCBUG:365989

diff --git a/shell/desktopview.cpp b/shell/desktopview.cpp
index 509184d..83866dc 100644
--- a/shell/desktopview.cpp
+++ b/shell/desktopview.cpp
@@ -193,7 +193,7 @@ bool DesktopView::event(QEvent *e)
     } else if (e->type() == QEvent::FocusIn) {     //FIXME: this should *not* be needed
         ensureWindowType();
 
-    } else if (e->type() == QEvent::FocusOut) {
+    } else if (e->type() == QEvent::FocusOut && containment()) {
         QObject *graphicObject = containment()->property("_plasma_graphicObject").value<QObject *>();
         if (graphicObject) {
             graphicObject->setProperty("focus", false);
